### PR TITLE
JENKINS-56109 Compatible CSS changes for tables to divs

### PR DESCRIPTION
**Pull request by timja**
See [JENKINS-56109](https://issues.jenkins-ci.org/browse/JENKINS-56109).
and https://github.com/jenkinsci/jenkins/pull/3895

Changes from tables to divs:
* reverted unnecessary white space changes
* commented out vertical-align middle in a couple of places, that will be re-applied on the tables to divs PR
* removed the border on draggable elements, not there on master and was probably only there as debug not intended to be there permanently.

<details>
<summary>Screenshots</summary>

image

image

screencapture-localhost-8080-jenkins-configure-2020-06-26-09_10_57

</details>

### Proposed changelog entries

* Entry 1: JENKINS-56109, Minor styling tweaks in preparation for changing form layout
* ...

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: war/src/main/less/base/style.less
Comment (fqueiruga) [inline war/src/main/less/base/style.less:1393]: This one is not looking good for `.dropdownList-container`
Captura de pantalla 2020-06-26 a las 10 20 55

File: war/src/main/js/widgets/config/tabbar.less
Comment (fqueiruga) [inline war/src/main/js/widgets/config/tabbar.less:183]: we need to keep this rule, otherwise things will get too packed

**Review by fqueiruga — CHANGES_REQUESTED** [review]

File: war/src/main/js/widgets/config/tabbar.less
Comment (fqueiruga) [reply war/src/main/js/widgets/config/tabbar.less:183]: Really open  for suggestions though

Comment (timja) [reply war/src/main/js/widgets/config/tabbar.less:183]: isn't that moved to line 186?

File: war/src/main/js/widgets/config/tabbar.less
Comment (fqueiruga) [inline war/src/main/js/widgets/config/tabbar.less:233]: These rules should be back in, otherwise it's tough to properly appreciate the nesting:

Captura de pantalla 2020-07-01 a las 9 19 09

File: war/src/main/js/widgets/config/tabbar.less
Comment (fqueiruga) [inline war/src/main/js/widgets/config/tabbar.less:239]: Ditto

**Review by fqueiruga — CHANGES_REQUESTED** [review]

**Comment by timja** [comment]
If you have time would be great if you could PR the adjustments, otherwise I’ll try get to it before the end of the week

**Comment by fqueiruga** [comment]
I'll try to sneak those in on Thursday

File: war/src/main/js/widgets/config/tabbar.less
Comment (timja) [reply war/src/main/js/widgets/config/tabbar.less:233]: Re-applied:

image

File: war/src/main/js/widgets/config/tabbar.less
Comment (timja) [reply war/src/main/js/widgets/config/tabbar.less:183]: done

**Review by fqueiruga — APPROVED** [review]
LGTM. There are some items that need to be polished but I think this one should be merged and create follow up PRs.

**Comment by timja** [comment]
@jenkinsci/core-pr-reviewers could someone take a look please

**Review by oleg-nenashev — APPROVED** [review]
I do not see anything wrong with the code. Testing manually to verify the layout

**Review by oleg-nenashev — APPROVED** [review]
I have tested a number of pages using https://github.com/oleg-nenashev/demo-jenkins-config-as-code with the patched Jenkins core. There are some minor artifacts here and there, but overall the UI looks pretty well after the changes. I do not think the issues block the merge, but it would be nice to get them solved by the .1 LTS

Tested pages:

* /configure
* /configureSecurity/
* /configureTools/
* Agent configuration page
* Cloud configuration
* Job configuration
* Folder configuration
* Job configuration


## A few discovered glitches

Project configuration - odd vertical indenting, different colors

image


Folder - Pipeline library config (the latter highligh is likely a "feature")

image

**Comment by fqueiruga** [comment]
Agree, those should be fixed. We just merged the PR to unblock it

**Comment by timja** [comment]
> Agree, those should be fixed. We just merged the PR to unblock it

Now or in a followup?

**Comment by fqueiruga** [comment]
follow up

**Comment by timja** [comment]
This PR is now ready for merge, we'll merge it after 24 hours if there's no negative feedback

**Comment by oleg-nenashev** [comment]
@jenkinsci/core  Maybe we could merge it earlier (by tomorrow's weekly) to facilitate feedback in weekly and fixing regressions earlier for the next LTS

**Comment by oleg-nenashev** [comment]
Actually we do not need to hurry, we are skipping tomorrow's weekly anyway

**Comment by jglick** [comment]
> Folder - Pipeline library config (the latter highligh is likely a "feature")

see discussion in https://issues.jenkins-ci.org/browse/JENKINS-62813

**Comment by oleg-nenashev** [comment]
@jenkinsci/core Any concerns about merging the change?

**Pull request merged**
